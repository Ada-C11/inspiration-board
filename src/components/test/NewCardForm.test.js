// src/components/test/NewPetForm.test.js
import React from 'react';
import NewPetForm from '../NewCardForm';
import { shallow } from 'enzyme';

describe('NewCardForm', () => {
  test('that it matches an existing snapshot', () => {
    // First Mount the Component in the testing DOM
    // Arrange
    const wrapper = shallow( <NewPetForm onSubmitNewCardCallback={() => {} } />);

    // Assert that it looks like the last snapshot
    expect(wrapper).toMatchSnapshot();
  });
});
