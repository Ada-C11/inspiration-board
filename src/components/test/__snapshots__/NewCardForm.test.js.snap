// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCardForm that it matches an existing snapshot 1`] = `
<form
  className="new-card-form"
  onSubmit={[Function]}
>
  <h3>
    Add a Inspirational Message!
  </h3>
  <label>
    Text:
    <input
      name="text"
      onChange={[Function]}
      type="text"
      value=""
    />
  </label>
  <label>
    Emoji:
    <select
      name="emoji"
      onChange={[Function]}
      value=""
    >
      <option
        value=""
      >
        Optional Emojis
      </option>
      <option
        value=""
      >
        None
      </option>
      <option
        value="heart_eyes"
      >
        Heart Eyes
      </option>
      <option
        value="beer"
      >
        Beer
      </option>
      <option
        value="clap"
      >
        Clap
      </option>
      <option
        value="sparkling_heart"
      >
        Sparkling Heart
      </option>
      <option
        value="heart_eyes_cat"
      >
        Cat Eyes Heart
      </option>
      <option
        value="dog"
      >
        Dog
      </option>
    </select>
  </label>
  <input
    className="btn btn-success new-card-form--submit"
    name="submit"
    type="submit"
    value="Add a Card"
  />
</form>
`;
